### DISCUSSION TITLE
How to add Text beside of an image?

### DISCUSSION THREAD

**Discussion by raysefo** [QuestPDF/QuestPDF#1052] [Q&A]
Hi,
I am trying to generate a header for the proforma invoice. Proforma Invoice text should be bold and bigger fonts and a throughout horizontal line below them. I couldn't manage so far. My code does not work. One more thing, Can we color the background like in the picture?

```
container.Row(row =>
{
    row.RelativeItem().Column(column =>
    {
        column.Item().Width(170).Image(LogoImage);
        //column.Item().MaxWidth(1000).LineHorizontal(1);

    });

    row.RelativeItem(4).Column(column =>
    {
        column.Item().AlignCenter().Text($"PROFORMA INVOICE").Bold().FontSize(20);
    });

    //row.ConstantItem(100).LineHorizontal(1);

});
```

![image](https://github.com/user-attachments/assets/9a81ec29-fda4-49b5-99e9-f63f45b98aa4)

**Comment by girlpunk** [comment]
> My code does not work

Please be more specific. Are there any error message or exceptions? How does the result of the code differ from the desired output?

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
